pkg/didcomm/protocol/didexchange/service.go

```diff
@@ -92,6 +94,12 @@ type context struct {
 	didStore           didstore.Storage
 }
 
+// opts are used to provide client properties to DID Exchange service
+type opts interface {
+	// PublicDID allows for setting public DID
+	PublicDID() string
+}
```

Comment:
It does not look like options, I would rather call it `actionDependency`

Reply:
I would prefer to leave it as opts since it is optional.

Reply:
ok